client/src/MainLayout.css

```diff
@@ -0,0 +1,42 @@
+.main-layout {
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+}
```

Comment:
Ahaa. Please use TailwindCSS instead of Pure CSS for better UI

Reply:
sure bhaiya-